FWCore/src/components/PileupOverlayAlg.cpp

```diff
@@ -26,7 +26,7 @@ StatusCode PileupOverlayAlg::initialize() {
   if (m_doShuffleInputFiles) {
     m_pileupFileIndex = static_cast<unsigned int>(m_flatDist() * m_pileupFilenames.size());
   }
-  m_reader.openFile(m_pileupFilenames[m_pileupFileIndex]);
+  m_reader.openFiles({m_pileupFilenames[m_pileupFileIndex]});
```

Comment:
Do we want to use the old interface for single files or this one?

Reply:
Here it doesn't really matter, no? actually the pileup infrastructure implements a simple version of TChain, so with this change it can be simplified in the future.

Reply:
Fair enough.